src/main/cpp/t06_homework.cpp

```diff
@@ -80,9 +80,72 @@
 
 #include "t06_homework.h"
 #include <iostream>
+#include <string>
+#include <set>
 
 using namespace std;
+string frase;
+set<string> arr;
+int j=0;
+int count_accent=0;
+void division() {
+int n=frase.length();
+string word, maxword;
+for (int i=0; i<n;i++){
+    if (frase[i] !=' '){
+        word=word+frase[i];}
+    else {
+        arr.insert(word);
+        j++;
+        word="";
+    };
+};
+arr.insert(word);
+j++;
+}
 
-int t06_homework() {
+string upper(string s){
+for (basic_string<char>::iterator p = s.begin();p != s.end(); ++p) {
+    *p = toupper(*p); 
+}
+return s;
+}
+void countaccent(string s){
+    count_accent=0;
+    for (basic_string<char>::iterator p = s.begin();p != s.end(); ++p) {
+        if (*p>='A' and *p<='Z') count_accent++; 
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Comparison of a boolean expression with an integer.](https://app.codacy.com/app/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4555431)